llpc/test/shaderdb/core/TestXfbStateMetadata.spvasm

```diff
@@ -0,0 +1,109 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+; SHADERTEST: define dllexport spir_func void @main() #2 !spirv.ExecutionModel !6 !lgc.xfb.state !7 !lgc.shaderstage !8
+; SHADERTEST: !7 = !{i32 0, i32 4, i32 -1, i32 0, i32 -1, i32 0, i32 -1, i32 0}
+; SHADERTEST-LABEL: {{^// LLPC}} transform feedback export info (vertex shader)
+; SHADERTEST: float (builtin = PointSize), xfbBuffer = 0, xfbStride = 4, xfbOffset = 0
+; SHADERTEST: AMDLLPC SUCCESS
```

Comment:
Add a SHADERTEST-NOT line to make sure that builtin = Position does not appear.

Reply:
Done.